src/libraries/System.Private.CoreLib/src/System/Text/Ascii.Equality.cs

```diff
@@ -363,6 +355,7 @@ private interface ILoader<TLeft, TRight>
             static abstract nuint Count256 { get; }
             static abstract Vector128<TRight> Load128(ref TLeft ptr);
             static abstract Vector256<TRight> Load256(ref TLeft ptr);
+            static abstract bool Compare256(ref TLeft left, ref TRight right);
```

Comment:
subjective: we could call it `EqualAndAscii` to better represent what it does

```suggestion
            static abstract bool EqualAndAscii(ref TLeft left, ref TRight right);
```